### PR TITLE
Refactor auth tests isolate scenarios

### PR DESCRIPTION
**Pull request by tainguyenbui**
## Description
Refactor Auth integration tests to become isolated, not having to depend on previous steps.

Increase dependabot dependencies check frequency

## Changes
- Refactor Auth tests
- Randomize inputs
- Create helper to generate a valid password 

## Checks
- [x] Project Builds
- [x] Project passes tests and checks
- [ ] Updated documentation accordingly

### CODE REVIEW

**Comment by tainguyenbui** [comment]
bot: integration

**Comment by tainguyenbui** [comment]
bot: integration

**Comment by tainguyenbui** [comment]
bot: integration

**Comment by tainguyenbui** [comment]
bot: integration

File: packages/framework-integration-tests/integration/providers/aws/auth.integration.ts [hunk 1]
```diff
@@ -243,28 +285,110 @@ describe('With the auth API', () => {
     })
 
     it('can query a secured read model they have privileges for', async () => {
+      const mockSku = random.alphaNumeric(random.number({ min: 10, max: 20 }))
+      const mockDisplayName = lorem.sentence()
+      const mockDescription = lorem.paragraph()
```
Comment (NickSeagull) [inline packages/framework-integration-tests/integration/providers/aws/auth.integration.ts:290]: Best usage of lorem ipsum i've ever seen

**Pull request merged**